### PR TITLE
Update runs/search with 4 new fields Refs. #1928

### PR DESCRIPTION
**Pull request by AirinB**
- updated runs/search with 4 new fields: start date, stop date, planned start, and planned stop Refs #1928

### CODE REVIEW

**Comment by AirinB** [comment]
@atodorov is this all that it needs to be done for the /runs/search/ page?

File: tcms/testruns/templates/testruns/search.html [hunk 1]
```diff
@@ -71,6 +71,28 @@
                 <input class="bootstrap-switch" id="id_running" name="running" type="checkbox" {% if form.running.value != "0" %}checked{% endif %} data-on-text="{% trans 'Yes' %}" data-off-text="{% trans 'No' %}">
             </div>
         </div>
+        <div class="form-group">
+            <label class="col-md-1 col-lg-1" for="id_start_date">{% trans "Start Date" %}</label>
```
Comment (atodorov) [inline tcms/testruns/templates/testruns/search.html:75]: Don't capitalize field names -> Start Date -> Start date

File: tcms/testruns/templates/testruns/search.html [hunk 2]
```diff
@@ -71,6 +71,28 @@
...
+            <div class="col-md-3 col-lg-3">
+                <input id="id_start_date" type="datetime-local" class="form-control">
+            </div>
+
+            <label class="col-md-1 col-lg-1" for="id_stop_date">{% trans "Stop Date" %}</label>
+            <div class="col-md-3 col-lg-3">
+                <input id="id_stop_date" type="datetime-local" class="form-control">
```
Comment (atodorov) [inline tcms/testruns/templates/testruns/search.html:82]: These fields don't open a calendar widget, see plan/search page for example.

File: tcms/testruns/templates/testruns/search.html [hunk 3]
```diff
@@ -71,6 +71,28 @@
...
+            </div>
+
+            <label class="col-md-1 col-lg-1" for="id_planned_stop">{% trans "Planned Stop" %}</label>
+            <div class="col-md-3 col-lg-3">
+                <input id="id_planned_start" type="datetime-local" class="form-control">
+            </div>
+        </div>
 
         <div class="form-group">
             <label class="col-md-1 col-lg-1" for="id_tag">{% trans "Tag" %}</label>
```
Comment (atodorov) [inline tcms/testruns/templates/testruns/search.html:98]: Move the Tag field after Plan ID (top-right) to use the available free space on screen and bring the results table upwards a little bit.

File: tcms/testruns/templates/testruns/search.html [hunk 4]
```diff
@@ -71,6 +71,28 @@
...
+        <div class="form-group">
+            <label class="col-md-1 col-lg-1" for="id_planned_start">{% trans "Planned Start" %}</label>
+            <div class="col-md-3 col-lg-3">
+                <input id="id_planned_start" type="datetime-local" class="form-control">
+            </div>
+
+            <label class="col-md-1 col-lg-1" for="id_planned_stop">{% trans "Planned Stop" %}</label>
+            <div class="col-md-3 col-lg-3">
+                <input id="id_planned_start" type="datetime-local" class="form-control">
+            </div>
```
Comment (atodorov) [inline tcms/testruns/templates/testruns/search.html:94]: All of the 4 fields should be tied to the search functionality on this page in order to be useful.

**Review by atodorov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**